app/src/main/java/org/mozilla/fenix/ext/Int.kt

```diff
@@ -5,8 +5,19 @@
 package org.mozilla.fenix.ext
 
 import android.content.Context
+import androidx.annotation.ColorInt
+import androidx.annotation.ColorRes
 import androidx.core.content.ContextCompat
 import org.mozilla.fenix.ThemeManager
 
+/**
+ * Returns the color resource corresponding to the attribute.
+ */
+@ColorRes
 fun Int.getColorIntFromAttr(context: Context): Int = ThemeManager.resolveAttribute(this, context)
+
+/**
+ * Returns the color int corresponding to the attribute.
+ */
+@ColorInt
```

Comment:
`Int` extension methods. These methods make me do a little sick in my mouth. 

I think I'd prefer them as `Context` methods. I appreciate you are making these _better_ by adding the annotations. Please file issues to make these better, then land this.

Reply:
Filed #4005